### DISCUSSION TITLE
clone

### DISCUSSION THREAD

**Discussion by Subfrom** [Merve40/ptt#12] [Q&A]
I clone can not install how to fix ?

**Comment by Merve40** [comment]
Do you mean you cannot install with `npm install` ? Please be more specific.

**Reply by Subfrom** [reply]
Yes npm install error

**Reply by Merve40** [reply]
There seems to be a problem with a dependency. I am looking into it now. I will push an update if I can find the problem

**Reply by Subfrom** [reply]
![image](https://user-images.githubusercontent.com/59534333/155177102-eea913c3-8d86-46a8-8590-b8f02a93fc43.png)
![image](https://user-images.githubusercontent.com/59534333/155177164-e5ae640e-d555-41f2-928d-700b7e29a294.png)
![image](https://user-images.githubusercontent.com/59534333/155177030-c0bfe8a5-81ed-4d30-871b-36b0ecee6aed.png)

**Reply by Subfrom** [reply]
How To Fix ?

**Reply by Merve40** [reply]
Unfortunately an important API I was using was removed. I am trying to find an alternative solution. As this project is **not** a high priority for me, it might take a while for me to push an update. However, you are welcome to contribute to this project :)

**Reply by Subfrom** [reply]
Ho Is okay Thank

**Reply by Merve40** [reply]
Hi Subfrom,
I have removed the broken dependencies and pushed the new version. You should be able to install it now.
